pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartServiceManagerCommand.java

```diff
@@ -192,7 +192,20 @@ public boolean execute()
   }
 
   private void startPinotService(Map<String, Object> properties) {
-    startPinotService(ServiceRole.valueOf(properties.get(PINOT_SERVICE_ROLE).toString()), properties);
+    ServiceRole role = ServiceRole.valueOf(properties.get(PINOT_SERVICE_ROLE).toString());
+    switch (role) {
+      // Broker and Server can be started in parallel always
+      case BROKER:
+      case SERVER:
```

Comment:
intentionally left controller out of here as it seemed there was some reason to. Otherwise, I'd like to put it in :D

Reply:
I dont think there is any reason to not add controller. @fx19880617

Reply:
If it's a new cluster, then controller has to be started first, otherwise broker and server will fail.

Probably we can add a check to decide if the cluster is already there then start controller in parallel as well.

Reply:
hmm so in this case, if you have multiple CLI args, you can be in the wrong order subtly.. maybe they need to be sorted?

Reply:
also I don't know how we check if "cluster is already" so if you feed me a command I can...

Reply:
If user starts broker/server first for a new cluster, then it will fail in-relevant to your changes.

Reply:
to check if a cluster is already there, you can create a spectator HelixManager by:
```
HelixManager spectatorHelixManager = HelixManagerFactory.getZKHelixManager(_clusterName, instance_id, InstanceType.SPECTATOR, _zkServers);
spectatorHelixManager.connect();
spectatorHelixManager.disconnect();
``` 

If it throws exception, then it means the cluster is not there.

Reply:
In the context of SM I don't know what to use as instance_id

Reply:
nm